src/me/mrCookieSlime/Slimefun/SlimefunGuide.java

```diff
@@ -716,7 +716,11 @@ public void run(final Player p) {
 							index++;
 						}
 						else {
-							menu.addItem(index, new CustomItem(Material.BARRIER, StringUtils.formatItemName(sfitem.getItem(), false), "", "&rYou do not have Permission", "&rto access this Item"));
+							if (Messages.local.getTranslation("messages.item-permission").get(1).isEmpty()) {
```

Comment:
That isEmptyCheck is really unneccessary, you should work with the list instead.
This is just bad design.

Reply:
This message key also needs to be updated.

Reply:
Done, not 100% sure if this works because I am not familiar with this, I never took java courses...

Reply:
That will definitely not work.
This is the totally wrong approach for this.

Reply:
Alright, I cannot continue on this then, I tried everything I can think of, I simply do not have enough knowledge about Lists. You can close the PR or if you can make it work that would be better but I know you are very busy so by all means just close it. Sorry for wasting your time, any other PR's I do will be of things I do know, I just need to take a couple lessons about lists.

Reply:
Alright, what about now?

Reply:
No no no no no. Don't do that.
You just have to call list.toArray(new String[list.size()]) to convert it to an Array.
But you will have to remove the empty lore line and include it in the messages.yml

Reply:
Alright, I figured it out I hope, I was dumb and forgot to check the various constructors CustomItem has.